DOCS-2487 / 27 / Docs 2487 preferences, access, ftp, and custom app UI changes (#4513)
truenas/documentation: content/SCALE/TopToolbar/Settings/_index.md

{{< include file="/static/includes/ChangeLoggedInUserPassword.md" >}}

## Preferences

The **Preferences** screen settings configure the color theme, session duration before timing out and logging out the currently logged-in user, and localization settings for the logged-in user account.

{{< trueimage src="/images/SCALE/Dashboard/PreferencesScreen.png" alt="Preferences Screen" id="Preferences Screen" >}}

### Customizing UI Theme and Localization

> The **Theme** settings, moved from the **System > General Settings > GUI** configuration screen, now show in the **Preferences** screen, which customizes the UI color theme for the session. for the currently logged-in user. 

Reviewer: There's an extra period and I don't think the narration of the change is needed?

```suggestion
The **Theme** settings customize the UI theme colors for the currently logged-in user. 
```

Options are: **ixDark** (default option), **ixBlue**, **Dracula**, **Nord**, **Paper**, **Solarized Dark**, **Midnight**, and **High Contrast**.
Selecting an option immediately changes the UI to the selected color theme.

> **Session Timeout**, moved from the **System > Advanced Settings > Access** configuration screen, sets the number of seconds a session remains active or inactive before it is automatically ended by logging out the user.

Reviewer: Same thought here. I don't think we're usually in the practice of explaining where a setting was moved from within the versioned documentation.

**Localization** sets the screen language, date format, and time format for the currently logged-in user account. To see the keyboard map and timezone, use the **System > General Settings > Localization** configuration screen.

Select the desired language for the UI from the **Language** dropdown list. The default setting is **English** but might be customized to the language based on the country of origin for an Enterprise customer. You can filter the list by typing in the field after clearing the default value, or use the scroll option to find and select a language.

Select the desired format in the **Date Format** and **Time Format** that matches your geographic location.

Click **Save** to set all changes for the currently logged-in user.

### Changing UI Session Timeout

TrueNAS automatically terminates the currently logged-in user session when the default session timeout expires and shows the TrueNAS login splash screen.

> To extend the allotted session time, go to the **Preferences** screen, and change the **Session Timeout** to a value that suits the use case for the logged-in user. 

Reviewer: ```suggestion
To extend the allotted session time, go to the **Preferences** screen and change the **Session Timeout** to a value that suits the use case for the logged-in user. 
```

{{< hint type=tip >}}
The default lifetime setting is 300 seconds or five minutes.

The maximum is 2147482 seconds, or converting it to hours/minutes/seconds, 596 hours, 31 minutes, and 22 seconds.

> If converting it to days/hours/minutes/seconds, 24 days, 20 hours, 31 minutes, and 22 seconds.

Reviewer: Having both translations one after another is confusing, what about:  
```suggestion
The maximum is 2147482 seconds, or 596 hours (24 days and 20 hours), 31 minutes, and 22 seconds.
```
